spring-cloud-gcp-dependencies/pom.xml

```diff
@@ -192,6 +192,12 @@
 				<artifactId>netty-tcnative-boringssl-static</artifactId>
 				<version>2.0.7.Final</version>
 			</dependency>
+
+			<dependency>
+				<groupId>ch.qos.logback.contrib</groupId>
+				<artifactId>logback-json-classic</artifactId>
+				<version>0.1.5</version>
```

Comment:
I don't think we should be managing the version this dependency for everyone in the BOM. It should just go into the dependency management section of the parent POM.